core/templates/pages/exploration-editor-page/services/exploration-property.service.ts

```diff
@@ -31,8 +31,8 @@ import { LoggerService } from 'services/contextual/logger.service';
   providedIn: 'root'
 })
 export class ExplorationPropertyService {
-  public displayed: string;
-  savedMemento: string;
+  public displayed: string | boolean;
+  savedMemento: string | boolean;
```

Comment:
This variable can have both string and boolean values depending on the derived class. Classes like ExplorationCategoryService stores string values and child classes like ExplorationCorrectnessFeedbackService store boolean values in this variable. So, the type has been fixed here.

By logging this variable, I found boolean values.

https://user-images.githubusercontent.com/24855641/147821000-c607a019-d1f2-4124-b140-f96183eeb106.mp4